openflow15/group.go

```diff
@@ -433,3 +433,110 @@ func (prop *GroupBucketPropWatch) UnmarshalBinary(data []byte) (err error) {
 
 	return
 }
+
+type NTRSelectionMethodType string
+
+const (
+	OFPGPT_EXPERIMENTER   = 0xFFFF
+	NTR_VENDOR_ID         = 0x0000154d
+	NTRT_SELECTION_METHOD = 1
+
+	NTR_HASH    NTRSelectionMethodType = "hash"
+	NTR_DP_HASH NTRSelectionMethodType = "dp_hash"
+)
+
+type NTRSelectionMethod struct {
+	Type             uint16
+	Length           uint16
+	ExperimenterID   uint32
+	ExperimenterType uint32
+	// padding with 4 bytes.
+	SelectionMethod [16]byte
+	SelectionParam  uint64
+	// Note, a valid field is supposed to configure with MatchField.HasMask=false, and OVS will apply its
+	// Value to the packet field as a mask.
```

Comment:
Was the comment copied from the original C code?

Reply:
The requirement is from OVS code, but the sentence is not directly copied from source code, "MatchField", "HasMask" are struct or property defined in libOpenflow

Reply:
How about adding single quote to surround them?

Reply:
updated.